docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -478,12 +478,12 @@ Here is an example of how to instantly deploy to created containers::
 .. note:: If you're reading the docs from the beginning, this may be the first example you've seen of an Ansible playbook. This is not an inventory file.
           Playbooks will be covered in great detail later in the docs.
 
-.. _using_multiple_inventory_sources:
+.. _using_multiple_inventory_directories:
 
-Using multiple inventory sources
-++++++++++++++++++++++++++++++++
+Using multiple inventory directories
+++++++++++++++++++++++++++++++++++++
 
-You can use multiple inventory sources if you want to target multiple environments or reuse common
+You can use multiple inventory directories if you want to target multiple environments or reuse common
```

Comment:
this is not really limited to directories, the original sources was correct

Reply:
Fixed